_static/css/furo.css

```
@@ -0,0 +1,2399 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

We should mark what are those files from and paste info about a license

```suggestion
/*!
This file comes from `furo` package
Copyright (c) 2020 Pradyun Gedam <mail@pradyunsg.me>
SPDX-License-Identifier: MIT
normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css
*/
```

---

I just add LICENSE file, name of program and version is named in commit message